sdk/keyvault/azure-security-keyvault-keys/README.md

```diff
@@ -14,7 +14,7 @@ Maven dependency for the Azure Key Vault Key client library. Add it to your proj
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-keys</artifactId>
-    <version>4.2.1</version>
```

Comment:
why was this ? 4.2.1 previously  on Master ?
Isn't master on beta ?

Reply:
Same comment about the READMEs.

Reply:
okay, its fine for README to point to stable release.
But the master pom should reflect beta. Else this will create confusions for both users and developers in our team when looking at the code.